src/AutoBuild.jl

```diff
@@ -1522,8 +1500,9 @@ function build_project_dict(name, version, dependencies::Array{Dependency})
     end
     # Always add Libdl and Pkg as dependencies
     stdlibs = isdefined(Pkg.Types, :stdlib) ? Pkg.Types.stdlib : Pkg.Types.stdlibs
-    project["deps"]["Libdl"] = first([string(u) for (u, n) in stdlibs() if n == "Libdl"])
-    project["deps"]["Pkg"] = first([string(u) for (u, n) in stdlibs() if n == "Pkg"])
+    project["deps"]["Libdl"] = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
```

Comment:
Maybe there should be compat info added on JLLWrappers?

Reply:
We always want the latest version of JLLWrappers, I think, because we're going to use that as the new way to "regenerate" JLL packages; since JLLWrappers is a bunch of macros that generate the JLLs at compile time, when we tag a new version of JLLWrappers, we will upgrade all the JLLs.